### PR TITLE
Add guidance for PHP strings containing apostrophes

### PR DESCRIPTION
**Pull request by rafecolton**

### CODE REVIEW

**Review by thienlnam — APPROVED** [review]
Team always double quotes "<3"

**Review by amyevans — APPROVED** [review]
👍 It's the little things in life 😄

File: php.md [hunk 1]
```diff
@@ -848,7 +848,17 @@ $test = 5;
 
 ### String
 
-- Always use single quotes `'` if no variables are needed.
+- Always use single quotes `'` if no variables or apostrophes are needed.
+- For strings that require apostrophes, prefer enclosure in double quotes over escaping. That makes it easier to search for the string in code.
```
Comment (flodnv) [inline php.md:852]:
```suggestion
- For strings that require apostrophes, we SHOULD prefer enclosure in double quotes over escaping. That makes it easier to search for the string in code.
```

Perhaps worth emphasizing that it's optional

**Review by iwiznia — APPROVED** [review]

File: php.md (hunk 1 above)
Comment (rafecolton) [reply php.md:852]: I think I would rather go in the other direction and make it required. I can't think of a good reason not to do this

Comment (flodnv) [reply php.md:852]: I don't like to require style rules that aren't enforced by a style fixer

Comment (hayata-suenaga) [reply php.md:852]: is it possible to add a rule for this enforce this with a style fixer?

Comment (rafecolton) [reply php.md:852]: Good question, I will see how involved that is and either add it or accept Flo's suggestion 😄

Comment (rafecolton) [reply php.md:852]: Ok, I spent 2.5 minutes searching and now I am committing Flo's suggestion

**Review by iwiznia — APPROVED** [review]

**Review by cristipaval — APPROVED** [review]

File: php.md (hunk 1 above)
Comment (flodnv) [reply php.md]: Haha, I also looked and failed to find it

**Review by flodnv — APPROVED** [review]

**Pull request merged**